src/main/java/edu/usf/cutr/gtfs_realtime/bullrunner/GtfsRealtimeProviderImpl.java

```diff
@@ -246,6 +262,7 @@ private void refreshTripVehicle() throws IOException, JSONException {
                 vpBuilder.setTimestamp(
                         Instant.parse(vehicleObj.getString(FIELD_LAST_UPDATED)).getEpochSecond()
                 );
+                vpBuilder.hasVehicle();

```

Comment:
I believe this line of code can be removed - it doesn't appear to do anything.

Reply:
I added this by mistake.